src/components/RefinementList.vue

```diff
@@ -17,6 +17,7 @@
       :createURL="state.createURL"
       :is-from-search="state.isFromSearch"
       :can-refine="state.canRefine"
+      :send-event="state.sendEvent"
```

Comment:
do we still need to attach the default sendEvent, or is that done in the InstantSearch connector?

Reply:
@Haroenv You mean calling the function when refining? If that's what you mean, it's done in the InstantSearch.js connector's refine and we're just exposing them as to the slot so that users can run it for custom purpose.